tutorial/branches/merge-to-production.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Merge to production
+permalink: /tutorial/branches/merge-to-production/
+---
+
+In the previous section you learned about [conflicting changes and how to resolve them](/tutorial/branches/simultaneous-changes/). Now you're finally at the point where you merge the branch to production. You can either do full or partial merge. Let's start with a partial merge. 
+
+## Partial merge
+
+First, let's examine the project diff further. 
+
+{: .image-popup}
+![Screenshot - Match Change from Production](/tutorial/branches/figures/project-diff-after-reset.png)
+
+You can see there are checkboxes to the left of each configuration in the list. This allows you to merge one a subset of the changes. So let's say we only want to merge the HTTP extractor change regarding bitcoin transactions. Uncheck all the checkboxes except the one near the **"Bitcoin" http extractor**.
+
+*Note: **Don't click the merge button** when you click Merge to production in the next step unless you uncheck the checkbox. If you accidently do you'll need to recreate the whole branch.*  
+
+Click **Merge to production** and in the modal uncheck the *Delete current development branch after merge.* checkbox. Only then click the **Merge** button.
+
+{: .image-popup}
+![Screenshot - Match Change from Production](/tutorial/branches/figures/partial-merge-dialog.png)
+
+When you do a progress bar will show up informing you of the progress of the merge. When the merge is finished, you will see only 3 changed configurations in your branch.
+
+{: .image-popup}
+![Screenshot - Match Change from Production](/tutorial/branches/figures/partially-merged-branch.png)
+
+Switch to production and examine the HTTP extractor configuration. Notice that a new version has been created with the merge message as change description. 
+
+{: .image-popup}
+![Screenshot - Match Change from Production](/tutorial/branches/figures/merged-http-ex-version.png)
+
+If you go to **Storage**, you'll see that the bucket used by this configuration still only has the `bitcoin_price` table and the `bitcoin_transactions` table is missing, even though you had it in your branch and you merged the config. This is expected. Branch storage is completely isolated and no data are merged back to production. You need to run the extractor in production to get the data in production **Storage**.
+
+While you're in the **Storage** section, examine that the branch buckets are still available if you toggle the switch to show development branch buckets. 
+
+### Full merge
```

Comment:
otázka jestli tohle má smysl nějak rozlišovat - v UI žádnej partial a full merge není, prostě mergneš vybrané konfigurace a buď branch smažeš nebo ne (což s vybranýma konfiguracema nijak nesouvisí).

Reply:
Přijde mi, že jo. V principu mi to přijde fajn zmínit takhle, protože tím získáváš další "tool" na to jak s tím pracovat. Jen proto, že to má nadpis tak to nemusí být v UI, ne?

Reply:
No jo a ne, teď to je tak, že na začátku říkáš jsou dva způsoby jak se to dá udělat a ten první je ...... .. . .. . . . .. a ten druhej je vlastně stejnej. Jako asi má smysl říct, že to jde udělat tak nebo tak (full merge má řekněme svoje výhody v tom, že víš, že jsi na nic nezapomněl). ale otázka je jestli to stojí za nadpis, když taková operace v reálu neexistuje.
Nicméně tohle je určitě není nic zásadního a moc bych se tím netrápil.